acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -46,4 +48,8 @@ public Duration getPollingInterval() {
   public String getDownstreamUrl() {
     return String.format(HTTP_URL_FORMAT, getHostname(), getTcpPort());
   }
+
+  public String getGenesisFile() {
```

Comment:
why does this get a "get" prefix - but many other properties do not?

Reply:
Consistency with NodeConfiguration (which is never used / referenced in the actual Acceptance Tests)